.github/workflows/ci-native.yml

```diff
@@ -0,0 +1,12 @@
+name: CI - Native
+
+on: workflow_call
+
+
+jobs:
+    build-test-sgx:
+      name: Build and test sgx with Docker
+      uses: ./.github/workflows/ci-build-test-reusable.yml
+      with:
+        version_name: "native"
```

Comment:
as an outsider i don't clearly understand why we are building "with docker" but also with version "native" (in my head native means native to your target platform, which docker abstracts away)

i'm actually a bit confused by what is `native` and why it needs a tracer as well. maybe a short README.md  or comment on the build process  / this could be helpful?  @Brechtpd

Reply:
we have single docker image supports all native/sgx (future r0 & sp1), and native one will be used by client for reference check.

Reply:
`native` just means run the block builder raw without any proofs, and trace is for debugging which prints out the revm trace inside raiko. This is just used to compare the trace emitted by client cuz some of them doesn't match up